Matchismo/Model/Card.h

```diff
@@ -0,0 +1,23 @@
+//
+//  Card.h
+//  Matchismo
+//
+//  Created by Yeliena Khaletska on 28.05.2024.
+//
+
+@import Foundation;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface Card : NSObject
+
+@property (strong, nonatomic) NSString *contents;
+
+@property (nonatomic, getter=isChosen) BOOL chosen;
+@property (nonatomic, getter=isMatched) BOOL matched;
+
+-(int)match: (NSArray *)otherCards;
```

Comment:
```suggestion
-(int) match:(NSArray *)otherCards;
```

Reply:
No space needed according to style guide so I removed both 😊